### ISSUE TITLE
Preverbs unrecognized on some VII verbs?

### ISSUE THREAD

**Issue by aaronfay** [UAlbertaALTLab/cree-intelligent-dictionary#574]
When I search itwewina.dev for [`ta-pê-miyonâkwaniyiw`](https://sapir.artsrn.ualberta.ca/cree-dictionary/search?q=ta-p%C3%AA-miyon%C3%A2kwaniyiw), I see results: 

![screenshot 2020-12-03 at 11 31 43](https://user-images.githubusercontent.com/1033941/101078522-2856b800-355b-11eb-9d9c-70cc37e34d3a.png)

But when I search for [`ta-pê-kiwêmakaniyiw`](https://sapir.artsrn.ualberta.ca/cree-dictionary/search?q=ta-p%C3%AA-kiw%C3%AAmakaniyiw) I don't get any results: 

![screenshot 2020-12-03 at 11 32 16](https://user-images.githubusercontent.com/1033941/101078571-373d6a80-355b-11eb-956c-5e190a0d5339.png)

You can find an example of this word at roughly 2:35 of [this video](https://creeliteracy.org/2020/01/23/a-j-felix-in-his-own-words-y-dialect/). I suspected this may be related to https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/issues/573, [but even without the obviate suffix the preverbs are still not recognized with that word](https://sapir.artsrn.ualberta.ca/cree-dictionary/search?q=ta-p%C3%AA-kiw%C3%AAmakan).

**Comment by aarppe** [comment]
The primary error was some VIIn's which had been misclassified in the CW source as VIIw, cf.

`kîwêmakan:kîwêmakan3 VIIw ;`

This has now been corrected in the FST and the dictionary XML. When the form was misclassified, most if not all analyses would have been incorrect, but now the FST says the following:

```
hfst-lookup -q src/analyser-gt-norm.hfst
ta-pe-kiwemakaniyiw	PV/ta+PV/pe+kîwêmakan+V+II+Ind+4Sg	0.000000
```
So what is needed is updating this to itwêwina as well.

**Comment by eddieantonio** [comment]
Fixed as of 95a2fdd 🎉

![Searching for ta-pê-kîwêmakaniyiw](https://user-images.githubusercontent.com/2294397/101701500-467d5600-3a3c-11eb-83f5-c862b7f960ed.png)

See: https://sapir.artsrn.ualberta.ca/cree-dictionary/search?q=ta-p%C3%AA-kiw%C3%AAmakaniyiw
